src/core/ugbio_core/vcf_utils.py

```diff
@@ -134,6 +189,82 @@ def intersect_with_intervals(self, input_fn: str, intervals_fn: str, output_fn:
         """
         self.__execute(f"gatk SelectVariants -V {input_fn} -L {intervals_fn} -O {output_fn}")
 
+    def view_vcf(
+        self,
+        input_vcf: str,
+        output_vcf: str,
+        n_threads: int = 1,
+        extra_args: str = "",
+    ) -> None:
+        """View/subset VCF file using bcftools view
+
+        Parameters
+        ----------
+        input_vcf : str
+            Input VCF file path
+        output_vcf : str
+            Output VCF file path
+        n_threads : int, optional
+            Number of threads to use (default is 1)
+        extra_args : str, optional
+            Additional arguments to pass to bcftools view (default is empty string)
+        """
+        # Build the bcftools view command
+        cmd_parts = ["bcftools", "view"]
+
+        # Add threads
+        cmd_parts.extend(["--threads", str(n_threads)])
+
+        # Add extra arguments if provided
+        if extra_args:
+            cmd_parts.extend(extra_args.split())
+
+        # Add output format and files
+        cmd_parts.extend(["-O", "z", "-o", output_vcf, input_vcf])
+
+        # Execute the view command
+        self.__execute(" ".join(cmd_parts))
+
+    def remove_filter_annotations(self, input_vcf: str, output_vcf: str, n_threads: int = 1) -> None:
```

Comment:
```suggestion
    def clear_filter_annotations(self, input_vcf: str, output_vcf: str, n_threads: int = 1) -> None:
```
as this function unfilters all records.